x/auth/vesting/README.md

```diff
@@ -4,6 +4,10 @@ sidebar_position: 1
 
 # `x/auth/vesting`
 
+:::warning
```

Comment:
Users still need to register the module to use existing vesting accounts

Reply:
Going point, I'll add.